docs/04-kubernetes.md

```diff
@@ -1 +1,41 @@
 # Kubernetes
+
```

Comment:
I would add a requirements section that lists what we expect them to understand and be able to do.